src/pages/home-page/HomePage.js

```diff
@@ -1,7 +1,14 @@
 import "./home-page.scss";
+import "./../../style/common/common.scss";
+import logo from "../../assets/logo.png";
+import LoginForm from "../../components/users/LoginForm/LoginForm";
 
 const HomePage = () => (
   <div className="home-page">
```

Comment:
Do you think you can chage some of these <div> tags with another semantic HTML tag? Like <section> for example?

Reply:
Not necessarily, the main goal is to have the more important parts of the main content well indicated which would help with the document structure and also facilitate the use of assistive technologies (like screen readers).

Tags like `section` are usually known as "[html landmarks](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#content_sectioning)".
If you change it to a section I would also recommend to have a header which in this case it could be hidden in case is not included in the design. This header is what a screen reader will use to navigate section and indicate to the user each one.

Reply:
> Do I have to change all divs by sections?

Nope, just the main content, it's not ok to overuse section either. And I just mentioned 'section' as an example, if you want to use 'header', 'article', 'main' is up to you and the content!